src/TcBlack/TcPou.cs

```diff
@@ -45,6 +45,35 @@ public TcPou Format()
             return this;
         }
 
+        public TcPou FormatImplementation()
+        {
+            uint indents = 0;
+            XmlNodeList nodes = doc.SelectNodes(".//Implementation/ST");
+            foreach (XmlNode node in nodes)
+            {
+                string formattedCode =
+                    new ImplementationCode(node.InnerText, Indentation, LineEnding)
+                    .Tokenize()
+                    .Format(ref indents);
+                string firstLine;
+                if (
```

Comment:
This is something I've started with my own projects. The first line of the implementation is always a comment `//` or empty. This is because when doing git diffs on the xml it's easier to read if the first code line is not attached to the `<![CDATA[`. This is separating the TwinCAT markup vs the actual user code.

For example
```
<Implementation>
        <ST><![CDATA[
F_CreateHashTableHnd(ADR(table), SIZEOF(table), hTable);
]]></ST>
      </Implementation>
```
vs

```
<Implementation>
        <ST><![CDATA[F_CreateHashTableHnd(ADR(table), SIZEOF(table), hTable);
]]></ST>
      </Implementation>
```